opensearchpy/_async/http_aiohttp.py

```diff
@@ -301,9 +297,9 @@ async def perform_request(
                 timeout=timeout,
                 fingerprint=self.ssl_assert_fingerprint,
             ) as response:
-                if is_head:  # We actually called 'GET' so throw away the data.
+                if is_head:
                     await response.release()
-                    raw_data = ""
+                    raw_data = ""  # HEAD method has no response body
```

Comment:
Should be `None`? There other other places where we assign things to `""` where in fact we mean none I believe, too.